docs/docs/basic-features/managing-segments.md

```diff
@@ -240,3 +240,30 @@ More simply, the order of precedence is:
 1. Identity
 2. Segment
 3. Flag
+
+## SDK Compatibility
+
+### `In` Segment operator
```

Comment:
Hmm... I actually meant more that we should create a section for the `IN` operator with the other operators, e.g. the Modulo operator has it's own section [here](https://github.com/Flagsmith/flagsmith/pull/3371/files#diff-5c8f900088e9b7550b4df2e8d2db0ad8fc3f9c1c3c99e295688043f7cad924ebR190). I think we should add a new section for the IN operator and define the compatibility there.

Reply:
OK Yep - have shifted some other stuff around its more logically laid out now.